src/test/java/unit/tests/testng/CartTest.java

```diff
@@ -0,0 +1,35 @@
+package unit.tests.testng;
+
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+import shop.Cart;
+import shop.RealItem;
+
+import static org.testng.Assert.assertEquals;
+
+public class CartTest {
+
+    private Cart cart;
+
+    @BeforeTest(groups = {"cart_tests"})
```

Comment:
Be careful with groups for before/after hooks. If hook belongs to group, then it will be executed only for this group. Attribute "alwaysRun" for annotation can help to avoid that behaviour. Please read description for this attribute to make sure that you understand what I am saying